logistics/scripts/modules/tools/ToolsHelper.psm1

```diff
@@ -256,7 +256,11 @@ function Invoke-DbDeploy {
     }
     $databaseType = $databaseIdLookup[$Database]
 
-    $tool = (Join-Path (Get-ToolsPath) 'EdFi.Db.Deploy')
+    $toolsPath = $ToolsPath
+
+    if([string]::IsNullOrEmpty($ToolsPath)){ $toolsPath = Get-ToolsPath }
```

Comment:
does it make sense to set the parameter to default to the Get-ToolsPath function?